examples/vision/md/grad_cam.md

```diff
@@ -171,30 +177,28 @@ Predicted: [('n02504458', 'African_elephant', 0.8871446)]
 ## Create a superimposed visualization
 
 
+
 ```python
-# We use cv2 to load the original image
-img = cv2.imread(img_path)
+# We load the original image
+img = keras.preprocessing.image.load_img(img_path)
+img = keras.preprocessing.image.img_to_array(img)
 
-# We resize the heatmap to have the same size as the original image
-heatmap = cv2.resize(heatmap, (img.shape[1], img.shape[0]))
+# We convert the heatmap to image
+heatmap = np.expand_dims(heatmap, axis=-1)
+heatmap = keras.preprocessing.image.array_to_img(heatmap)
 
-# We convert the heatmap to RGB
-heatmap = np.uint8(255 * heatmap)
+# We resize the heatmap to match image size
+heatmap = heatmap.resize((img.shape[1], img.shape[0]))
 
-# We apply the heatmap to the original image
-heatmap = cv2.applyColorMap(heatmap, cv2.COLORMAP_JET)
+# Show image overlayed with the heatmap
+plt.imshow(img / 255.0)
 
 # 0.4 here is a heatmap intensity factor
-superimposed_img = heatmap * 0.4 + img
-
-# Save the image to disk
-save_path = "elephant_cam.jpg"
-cv2.imwrite(save_path, superimposed_img)
-
-display(Image(save_path))
+plt.imshow(heatmap, cmap="jet", alpha=0.4)
```

Comment:
IMO the visuals from calling `display(Image(save_path))` are much better than using `plt`. Nevertheless, it should be possible to remove the cv2 dependency while still using `display(Image())`. We could do use `array_to_img` to create a PIL image holding the superimposed images, then use `img.save(path)` to save it to disk.

Reply:
I'll make the changes and get back to you soon. Thanks for reviewing this PR.

Reply:
Should I use something like a more minimal version of [this](https://gist.github.com/jimfleming/c1adfdb0f526465c99409cc143dea97b) to apply jet color map on the heatmap for RGB format basis `matplotlib.cm` and then use `img = img + 0.4 * heatmap`?

Reply:
I haven't tried it, and I image there might be a simpler solution, but you could do something like

```python
img = keras.preprocessing.image.load_img(img_path)
img = keras.preprocessing.image.img_to_array(img)
heatmap = np.expand_dims(heatmap, axis=-1)
heatmap = heatmap.resize((img.shape[1], img.shape[0]))
heatmap = keras.preprocessing.image.array_to_img(heatmap)
heatpmap = keras.preprocessing.image.img_to_array(heatpmap)
superimposed_img = heatmap * 0.4 + img
superimposed_img = keras.preprocessing.image.array_to_img(superimposed_img)
superimposed_img.save(...)
Display(Image(...))
```

Reply:
I was trying something like:

```python3
# We rescale heatmap to a range 0-255
heatmap = np.uint8(255 * heatmap)

# Colorize the heatmap to jet 
jet = cm.get_cmap('jet')
jet_colors = jet(np.arange(256))
jet_heatmap = jet_colors[heatmap]

# We create an image from the heatmap
jet_heatmap = keras.preprocessing.image.array_to_img(jet_heatmap)
jet_heatmap = jet_heatmap.resize((img.shape[1], img.shape[0]))
jet_heatmap = keras.preprocessing.array_to_img(jet_heatmap)

# Create an overlay of the heatmap on original image
superimposed_img = jet_heatmap * 0.4 + img

superimposed_img = keras.preprocessing.image.array_to_img(superimposed_img)
superimposed_img.save(...)
Display(Image(...))
```

Which gives me something like this:
![GRAD CAM with JET](https://github.com/keras-team/keras-io/raw/master/examples/vision/img/grad_cam/grad_cam_10_0.jpeg)
)

Reply:
> I haven't tried it, and I imagine there might be a simpler solution, but you could do something like
> 
> ```python
> img = keras.preprocessing.image.load_img(img_path)
> img = keras.preprocessing.image.img_to_array(img)
> heatmap = np.expand_dims(heatmap, axis=-1)
> heatmap = heatmap.resize((img.shape[1], img.shape[0]))
> heatmap = keras.preprocessing.image.array_to_img(heatmap)
> heatpmap = keras.preprocessing.image.img_to_array(heatpmap)
> superimposed_img = heatmap * 0.4 + img
> superimposed_img = keras.preprocessing.image.array_to_img(superimposed_img)
> superimposed_img.save(...)
> Display(Image(...))
> ```

This is giving me something like this:
![GRAD CAM without Jet cmap](https://user-images.githubusercontent.com/10649800/81509430-3d060100-9328-11ea-8817-ca17a86dc4e9.jpeg)
As the grayscale heatmap is directly being used superimposed without application of the Jet colormap.